### PR TITLE
KL-444: fix service worker error

### PR DESCRIPTION
**Pull request by Boltman92**

### CODE REVIEW

File: apps/extension/content-script.ts [hunk 1]
```diff
@@ -6,42 +6,52 @@ import { runtime } from 'webextension-polyfill';
...
-    return Promise.resolve();
-  }
+  return myPort;
+}
+
+connectPort();
+
+const CONTENT_SCRIPT = 'oko-contentscript';
+const INPAGE = 'oko-inpage';
+const PROVIDER = 'oko-provider';
```
Comment (MuKoJla) [inline apps/extension/content-script.ts]: maybe we can move these variables in one place?
Снимок экрана 2023-01-06 в 11 45 29
Снимок экрана 2023-01-06 в 11 45 08
Снимок экрана 2023-01-06 в 11 44 54

File: apps/extension/content-script.ts [hunk 2]
```diff
@@ -6,42 +6,52 @@ import { runtime } from 'webextension-polyfill';
...
+
+const pageStream = new WindowPostMessageStream({
```
Comment (MuKoJla) [inline apps/extension/content-script.ts]: or maybe it is also possible move `pageStream` to constants

File: apps/extension/inpage.ts [hunk 3]
```diff
@@ -1,16 +1,8 @@
-import { WindowPostMessageStream } from '@metamask/post-message-stream';
-
+import { inpageStream } from './src/constants/content-script';
 import { initializeProvider } from './src/dapp-connection-provider';
 
-const CONTENT_SCRIPT = 'oko-contentscript';
-const INPAGE = 'oko-inpage';
```
Comment (MuKoJla) [inline apps/extension/inpage.ts:6]: but maybe also be better to replace these too?
image
image

**Review by MuKoJla — DISMISSED** [review]

**Review by unixvb — DISMISSED** [review]

File: apps/extension/src/constants/content-script.ts [hunk 4]
```diff
@@ -0,0 +1,3 @@
+export const CONTENT_SCRIPT = 'oko-contentscript';
+export const INPAGE = 'oko-inpage';
+export const PROVIDER = 'oko-provider';
```
Comment (unixvb) [inline apps/extension/src/constants/content-script.ts]: why do you have 2 different variables with the same value?
`libs/ui/src/utils/dapp.utils.ts`

File: apps/extension/content-script.ts [hunk 5]
```diff
@@ -1,54 +1,56 @@
...
 import { runtime } from 'webextension-polyfill';
 
+import { CONTENT_SCRIPT, PROVIDER } from './src/constants/content-script';
+import { pageStream } from './src/constants/page-streams';
 import { DAppMessage } from './src/interfaces/dapp-message.interface';
 import { getWindowMetadata } from './src/utils/window.utils';
 
-const myPort = runtime.connect({ name: 'port-from-cs' });
+function connectPort() {
+  const myPort = runtime.connect({ name: 'port-from-cs' });
```
Comment (unixvb) [inline apps/extension/content-script.ts]: could you move `port-from-cs` into variable?

**Review by unixvb — DISMISSED** [review]

**Review by MuKoJla — DISMISSED** [review]

**Review by unixvb — APPROVED** [review]

**Review by MuKoJla — APPROVED** [review]

**Review by vladvn1905 — APPROVED** [review]

**Pull request merged**